### DISCUSSION TITLE
Facing error while sending reports to s3 bucket during security scanning (prowler V3.8.2)

### DISCUSSION THREAD

**Discussion by naguredington** [prowler-cloud/prowler#2762] [Q&A] [answered]
(venv-assessment) [cloudshell-user@ip-10-2-81-134 Python-3.9.16]$ prowler aws -b -M csv -B redtest123/redtestfolder01

Note: IAM user is having full admin rights & S3 bucket name also valid only but still facing error..

![image](https://github.com/prowler-cloud/prowler/assets/121438887/b0b0dfbd-dae6-4538-bbf4-64644ad38c17)

**Comment by jfagoagas** [comment, marked as answer]
Hi @naguredington, the `-B` option only accepts the bucket name, so you have to use the `-o/--output-directory` to specify a customer directory, like this:
`prowler aws -b -M csv -B redtest123 --output-directory redtestfolder01`

We are going to modify the documentation since it's not right.

Thanks for using Prowler 🚀

**Reply by naguredington** [reply]
Thanks @jfagoagas  ,its working now..

Also want to understands the output reports.. both CSV & HTML is not formed well in terms of format ( in CSV , lines are merged together & in HTML , few lines showing in vertical format & filter also not able to apply)

**Reply by jfagoagas** [reply]
Great!

Regarding the output formats that is strange, I think the better should be to send us the reports generated by your Prowler execution to review it because everything is fine in the latest release and everything should be in the right format.

**Reply by naguredington** [reply]
Hi @jfagoagas 

Please find the both html & csv files for your reference 
Note: ( In CSV , column lines are merged together & in HTML , few lines showing in vertical format & filter also not able to apply)

[Prowler outputs.zip](https://github.com/prowler-cloud/prowler/files/12425932/Prowler.outputs.zip)

**Reply by jfagoagas** [reply]
Hi @naguredington, I'm checking it:
- For the CSV format, everything is ok, what software are you using to open it? I've attached screenshots opening it with MacOS Numbers:
![Screenshot 2023-08-24 at 08 36 34](https://github.com/prowler-cloud/prowler/assets/16007882/6134212d-1e2a-4f17-880f-3cf640398ab8)
![Screenshot 2023-08-24 at 08 36 42](https://github.com/prowler-cloud/prowler/assets/16007882/f83c63ec-04d8-4fb4-98f1-90f690de1fe1)
- For the HTML format the same, opening it with Chrome I can see the results, navigate through the pages and use the filers:
![Screenshot 2023-08-24 at 08 37 58](https://github.com/prowler-cloud/prowler/assets/16007882/a0aa30b3-0559-4044-963c-91b30677ebbf)
![Screenshot 2023-08-24 at 08 38 15](https://github.com/prowler-cloud/prowler/assets/16007882/253f6dc6-ce44-48e1-bc31-d53fb1781d04)

**Reply by naguredington** [reply]
Hi @jfagoagas  ,

     I understood .. but i'm tried in different laptops (using windows OS & chrome browser ) .
     for me the result was not in right format.. please refer the attached below snap once !!
     one more thing V2 result was proper in our laptops when we test few months back only V3 outputs are issue.

 Note: ( In CSV , the first Row lines are merged together & in HTML ,  details/lines showing in vertical format & filter also not able to apply)
     
![image](https://github.com/prowler-cloud/prowler/assets/121438887/ab32722b-3b45-4d2e-93e7-501c0f804f08)

![image](https://github.com/prowler-cloud/prowler/assets/121438887/fcb8078f-487b-4f65-b62a-2e7ea29a96a8)

**Reply by jfagoagas** [reply]
I think I know what's happening with the CSV format. Your Excel is using the `,` as a delimiter instead of the `;` which is the delimiter used in that format from Prowler v3. So if you change the delimiter when opening/importing the CSV output it should be rendered right.

Regarding the HTML format, I'm not sure about what's happening, we have to dig in it more, maybe it'd be related with your screen resolution.

**Reply by naguredington** [reply]
Hi @jfagoagas  ,

      I have verified..
      As per ur advice ,changing Delimiter for CSV &  changing display resolution for html outputs is somehow working & able to see improvement in result view when we compare with previous one..
       But can u suggest me  (delimeter (,) & display resolution (150%) is already in recommended setup in windows machine , if i change these settings for prowler V3 result view alone , other normal activity works in windows machine getting tough to handle .. even if i want to see prowler V2 output again i have to change display resolution & delimeter configuration.. 
    Any permanent solution for windows machines ???
